src/redux/slice/modalSlice.js

```diff
@@ -14,8 +16,20 @@ export const modalSlice = createSlice ({
         closeModal: (state) => {
             state.isOpen = false;
         },
+        enableFollowButton: (state) => {
+            state.isFollowButtonEnabled = true;
+        },
+        disableFollowButton: (state) => {
+            state.isFollowButtonEnabled = false;
```

Comment:
mỗi twitter discord telegram đều riêng biệt e viết như này chỉ cần một cái flowed thì dểu hiển thị cái buton à e

Reply:
nên lưu giữ toàn bộ data trả về từ api getUser get social để sử lý trạng thái cảu button